css/css-masking/clip-path/WEB_FEATURES.yml

```diff
@@ -1,4 +1,6 @@
 features:
+- name: clip-path
```

Comment:
clip-path would be counting the tests from clip-path-boxes. Is this intended?

Reply:
I think this is OK, yeah. The tests for clip path boxes do use `clip-path`, so that's like a subfeature or aspect of the `clip-path` feature. I could imagine that clip-path-boxes is merged with clip-path in the future when enough time has passed and it doesn't make sense to describe clip path boxes as a separate feature.